Src/Plugins/Tests.Security/xunit.runner.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+  "maxParallelThreads": 1
```

Comment:
>maxParallelThreads [](http://example.com/codeflow?start=3&length=18)

Since the tests that uses httpClient share a common variable, we will need to run once at a time to prevent getting undesired results

Reply:
This is only needed during tests because we are injecting what we want to receive for each test